### PR TITLE
Added additional SEMPv2 resources

### PR DESCRIPTION
**Pull request by 195858**
Created new tasks for a number of SEMPv2 calls to facilitate DMR and VPN bridge setup

### CODE REVIEW

File: examples/solace_dmr.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    solace_dmr:
+      name: foo
+      state: absent
+  - name: Add 'foo' DMR Cluster
+    solace_dmr:
+      name: foo
+      state: present
+      settings:
+        tlsServerCertMaxChainDepth: 6
+        authenticationBasicPassword: Solace123!
```
Comment (mkst) [inline examples/solace_dmr.yml:14]: I see you removed passwords in a previous commit - here's one more

File: lib/ansible/module_utils/network/solace/solace_utils.py [hunk 2]
```diff
@@ -215,14 +243,15 @@ def _parse_bad_response(resp):
 
 def _make_request(func, solace_config, path, json=None):
     if func is requests.get:
-        path += '?count=' + str(MAX_REQUEST_ITEMS)
+        """ path += '?count=' + str(MAX_REQUEST_ITEMS) """
```
Comment (mkst) [inline lib/ansible/module_utils/network/solace/solace_utils.py:246]: was this deliberate? Do you have an alternative way of returning more than the default (10?) results?

File: examples/solace_dmr.yml (hunk 1 above)
Comment (195858) [reply examples/solace_dmr.yml:14]: I just use this as a random password. I did remove the parameter override for the Solace admin password so it's easier for a user who set up a docker container with the default admin user/pw to run the ansible tasks.
Happy to replace this with another placeholder that you prefer - you used "secret_password" in the queue example - shall I use that?

File: lib/ansible/module_utils/network/solace/solace_utils.py (hunk 2 above)
Comment (195858) [reply lib/ansible/module_utils/network/solace/solace_utils.py:246]: There are resources which do not support the "count" query parameter.
I made this configurable via a class variable instead. I'll send a new pull request.

**Comment by 195858** [comment]
Closing this request and creating a new request with updates as described.

**Pull request closed without merging**